modules/caddyhttp/rewrite/rewrite.go

```diff
@@ -88,6 +88,17 @@ type Rewrite struct {
 	// Performs regular expression replacements on the URI path.
 	PathRegexp []*regexReplacer `json:"path_regexp,omitempty"`
 
+	// If true, the rewrite will be forced to also apply to the
+	// query part of the URL. This is only needed if the configured
+	// URI does not include a '?' character which is normally used
+	// to determine whether the query should be modified. In other
+	// words, this allows rewriting both the path and query when
+	// using a placeholder as the replacement value, whereas otherwise
+	// only the path would be rewritten because the placeholder itself
+	// does not contain a '?' character. Only use this if the placeholder
+	// is trusted to not be vulnerable to query injections.
+	ModifyQuery bool `json:"modify_query,omitempty"`
```

Comment:
What if we call this `SpanComponents` or something a little more precise? It looks like what this change really does is make a single placeholder able to span URI components.

Reply:
That makes me think of a `<span>` JS component. That name doesn't bring any mental association to what it does to me.

Reply:
Hmm. I mean, obviously this is a backend context (a rewrite middleware) where we're talking about the URI... but I can see why you'd have that correlation.

What about `AllowExpansion` or something?